viewer/annotations/annotations.controller.js

```diff
@@ -21,7 +21,7 @@
         vm.createAnnotation = createAnnotation;
         vm.cancelNewAnnotation = cancelNewAnnotation;
 
-        vm.editedAnnotation = null; // Track which annotation is being edited right now
+        resetEditedValues();
```

Comment:
@jrchudy Did you mean to assign resetEditedValues to an instance of the controller here?